zulipterminal/cli/run.py

```diff
@@ -263,7 +263,8 @@ def main(options: Optional[List[str]]=None) -> None:
             prof.dump_stats("/tmp/profile.data")
             print("Profile data saved to /tmp/profile.data")
             print("You can visualize it using e.g."
-                  "`snakeviz /tmp/profile.data`")
+                  "`snakeviz /tmp/profile.data`"
+                  "NOTE: Use `pip install snakeviz` to install snakeviz")
```

Comment:
This *could* do a quick check to see if the command is present via running `snakeviz -h` and checking the result, if we wanted to smooth the user experience?